SPECS/nvidia-container-runtime/nvidia-container-runtime.spec

```diff
@@ -0,0 +1,63 @@
+Summary:      NVIDIA container runtime
```

Comment:
Does the license for the SPEC file allows us to take it?

Reply:
based on what i have seen from an email thread about this question we should be ok to use this.